src/main/java/Allez.java

```diff
@@ -0,0 +1,66 @@
+import java.util.Scanner;
+
+
+public class Allez {
+
+    public static void printList(Task[] list){
+        int count = 0;
+        while(list[count] != null){
+            System.out.println((count+1) + ". " + list[count].getStatusIcon() +
+                    " " + list[count].getDescription());
+            count+=1;
+        }
+    }
+
+//    public static void storeInList(String[] list, int count, String line) {
+//        list[count] = line;
+//    }
+    public static void main(String[] args) {
+
+        System.out.println("_________________________");
```

Comment:
Maybe can put all of these print statements in a separate function named greetingMessage to make the main function more readable